### DISCUSSION TITLE
Make "Delete Task" less prominent

### DISCUSSION THREAD

**Discussion by marbetschar** [elementary/tasks#223] [Ideas]
@elementary/ux is there a way to make the "Delete" button less prominent?

Right now it grabs a lot of attention during task editing:

![Delete](https://user-images.githubusercontent.com/392542/79224218-0341f780-7e5b-11ea-8e53-b91302750227.png)

**Comment by janxkoci** [comment]
What about a smaller button with just a trash icon? There is a similar button in the composer of Mail, although without the colour.
